### PR TITLE
German newsletter february 2022

### PR DESCRIPTION
**Pull request by wh0nix**
Hey guys,

here is the german version of the newsletter february 2022.
Sorry for beeing late, was a tough week!

### CODE REVIEW

File: content/blog/2022-03-05-newsletter.de.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+author: Anoxinon e.V.
+date: 2022-03-05
+categories: ['Newsletter']
+---
+
+_Diese Ausgabe wurde von [Anoxinon e.V.](https://anoxinon.de/) aus der XMPP Community übersetzt._
+
+Willkommen zum XMPP-Newsletter für den Februar 2022!
+
+Wir hoffen, dass du bisher einen guten Übergang in das neue Jahr hattest und dich auf die neue Ausgabe freust! Wir denken, dass diese Ausgabe über die Neujahrsfeiertage einen gewissen Umfang angenommen hat.
```
Comment (Echolon) [inline content/blog/2022-03-05-newsletter.de.md:12]: Irgendwas passt hier nicht. Das steht doch nicht so im Original?

**Review by Echolon — CHANGES_REQUESTED** [review]
See comment

File: content/blog/2022-03-05-newsletter.de.md (hunk 1 above)
Comment (wh0nix) [reply content/blog/2022-03-05-newsletter.de.md]: yep, you are right. That was from the newsletter before.
I made some changes.

**Review by Echolon — APPROVED** [review]
Thanks!

**Comment by Echolon** [comment]
@wh0nix you have push this to the feb2022 branch, not the master - can you change this?

**Pull request merged**